app/services/partners/request_approval_service.rb

```diff
@@ -0,0 +1,30 @@
+module Partners
+  class RequestApprovalService
+    include ServiceObjectErrorsMixin
+
+    def initialize(partner:)
```

Comment:
Is this a `::Partner` or a `Partners::Partner`? Can you leave a comment saying which